lib/themes/dosomething/paraneue_dosomething/scss/patterns/_card.scss

```diff
@@ -52,11 +52,6 @@
   overflow: hidden;
   background-color: $white;
 
-  .beta-card__video {
-    // The video element incorrectly sits in the beta card, quick hack to fix it. 
-    margin-bottom: -10px;
```

Comment:
@deadlybutter FYI don't need this anymore, but the issue wasn't a margin thing... you just needed to make the video display as block instead of the default of inline. Adding `display: block` also fixes this and in a slightly better way since it's not messing w/ margin which could affect other items. Just an FYI for future reference 😄